### PR TITLE
Rewrite QA instructions for setting up Normandy for testing

### PR DESCRIPTION
**Pull request by mythmon**
This is a complete rewrite of this document, so probably best to just check out the new version and not the diff. This along with a little hand waving, should cover "zero to addon testing".

@Osmose r?

### CODE REVIEW

File: docs/qa/docker.rst [hunk 1]
```diff
@@ -1,136 +1,111 @@
...
-The defaults for this are *usually* one of these two values:
-
-Linux with local Postgres
-  ``172.17.0.1``
-
-_`Docker Toolbox` with local Postgres (OSX or Windows)
-  ``192.168.99.1``
+Getting the code
+----------------
+We'll be using Docker Compose to run Normandy and the need services. The
```
Comment (Osmose) [inline docs/qa/docker.rst]: needed

File: docs/qa/docker.rst [hunk 2]
```diff
@@ -1,136 +1,111 @@
...
+repository ``normandy-compose`` contains the ``docker-compose.yml`` file to
```
Comment (Osmose) [inline docs/qa/docker.rst]: It'd be nice to link `normandy-compose` to the Github page.

**Comment by Osmose** [comment]
r+wc, nice work!

**Comment by mythmon** [comment]
Feedbacked ^

**Pull request merged**